SwiftPackageManager.md

```diff
@@ -44,6 +46,35 @@ in the `Build Settings` tab.
 
 <img src="docs/resources/SPMObjC.png">
 
+### Manually managing your `Package.swift` manifest
```

Comment:
Should be two `#`at the same level as "In Xcode".

Also, make it clearer this is an alternative as opposed to an additional step:

`## Alternatively, add Firebase by updating the `Package.swift` manifest:`

Reply:
Good catch - I actually meant to add 3 to the "In Xcode" bit since it's under the `## Installation` header. Let me know if you think they should both be 2.

Reply:
Agreed on 3